src/main/resources/templates/macros/page/buildinfo.vm

```diff
@@ -7,6 +7,9 @@
       #if($build_number)
         <th>Number</th>
       #end
+      #if($build_url)
+        <th>Link</th>
```

Comment:
if this is build url why not convert build number as link?

Reply:
See above, different semantics.